File: src/main/java/de/iteratec/iteraOfficeMap/workplace/WorkplaceRepository.java
package de.iteratec.iteraOfficeMap.workplace;

import org.springframework.data.jpa.repository.JpaRepository;

public interface WorkplaceRepository extends JpaRepository<Workplace, Long> {

    Workplace findByXEqualsAndYEqualsOrNameEquals(int x, int y, String name);

}

File: src/main/java/de/iteratec/iteraOfficeMap/workplace/WorkplaceRepository.kt
package de.iteratec.iteraOfficeMap.workplace

import org.springframework.data.jpa.repository.JpaRepository

interface WorkplaceRepository : JpaRepository<Workplace, Long> {

    fun findByXEqualsAndYEqualsOrNameEquals(x: Int, y: Int, name: String): Workplace?

}
